Logo fix

Kevin-Solomon opened the pull request:
# Logo Fix 
- Made all logo in navbar to redirect to index page
- Changed cards in wishlist and product page
![image](https://user-images.githubusercontent.com/80545228/154929891-b430d10b-0034-415b-9d94-3e6bd4b18521.png)

Kevin-Solomon committed:
add logo link to all navbar headers

Kevin-Solomon committed:
update cards in product and Wishlist page

Kevin-Solomon committed:
fix : html closing tag missing

AnkurChunekar commented on pages/login/login.html:
```diff
@@ -1,69 +1,73 @@
+          <input class="form-input" type="text" />
+          <label> Password </label>
+          <input class="form-input" type="text" />
+          <div class="help-container">
```
You can provide `id` and `for` attribute to input and label elements respectively.

AtulPant2704 reviewed (commented):
- You have not added link to index page in Home, Single Product and Profile page.
- You have also not added logo in Cart page.
- In Product page the Rating Filter radio buttons does not have name because of which all are getting clicked at the same time.

Kevin-Solomon committed:
fix : logo and form input

- Add logo in home page
- Add id and for attribute in form input
- Add name for radio button in product

Kevin-Solomon committed:
add link to navbar in profile and single product page

Kevin-Solomon committed:
fix : resolved conflicts

Kevin-Solomon committed:
fix : resolved conflicts

Kevin-Solomon committed:
fix : resolved conflicts

Kevin-Solomon committed:
Merge branch 'development' into logo-fix
